Mesh_3/include/CGAL/Mesh_3/Mesher_3.h

```diff
@@ -45,6 +45,7 @@
 #include <CGAL/Mesher_level_visitors.h>
 #include <CGAL/Kernel_traits.h>
 #include <CGAL/point_generators_3.h>
+#include <CGAL/assertions_impl.h>
```

Comment:
The `*_impl.h` headers are implementation details (for header-only) and should never be included explicitly: they are included by the non-impl headers.
```suggestion
#include <CGAL/assertions.h>
```